pkg/kubectl/cmd/get.go

```diff
@@ -203,3 +220,22 @@ func RunGet(f *cmdutil.Factory, out io.Writer, cmd *cobra.Command, args []string
 		return printer.PrintObj(r.Object, out)
 	})
 }
+
+func filterNonRunningPodsSelector(fieldSelector string) (fields.Selector, error) {
+	selector, err := fields.ParseSelector(fieldSelector)
+	if err != nil {
+		return nil, err
+	}
+	return selector, nil
+}
+
+func getSelector(args []string) string {
+	switch args[0] {
+	case "po", "pod", "pods":
```

Comment:
Can we share these at the top level?
Someone might add another alias for "pod" but wont know that we need to add it here as well.